### DISCUSSION TITLE
Error while deployment on Vercel

### DISCUSSION THREAD

**Discussion by Dhruv-2003** [wevm/wagmi#652] [Q&A]
I am using wagmi  for my Dapp and it is throwing same error multiple times while deployment . I tried the same thing with another Dapp and the issue comes out as same . I am attaching the SS of error below .
<img width="907" alt="error wagmi" src="https://user-images.githubusercontent.com/91938348/177030304-9c0fe298-d48d-4c3e-80a0-ddca3a5497be.png">

**Comment by tmm** [comment]
Looks like Vercel is erroring out because it doesn't have access to your repo, rather than a wagmi-specific issue.

**Reply by Dhruv-2003** [reply]
How can I resolve this error ? 
Because it seems my friend had same error while deploying his Dapp using Wagmi init , Please check and revert .

**Reply by tmm** [reply]
Do some searching around (e.g. https://github.com/vercel/vercel/issues/3728). This is not an issue with wagmi.
